modules/social_features/social_post/social_post.routing.yml

```diff
@@ -0,0 +1,64 @@
+post.overview_templates:
+  path: '/admin/structure/post'
+  defaults:
+    _controller: '\Drupal\Core\Entity\Controller\EntityListController::listing'
+    entity_type: 'post_template'
+    _title: 'Post templates'
+  requirements:
+    _permission: 'administer post entities'
+
+post.template_add:
+  path: '/admin/structure/post/template/add'
+  defaults:
+    _entity_form: 'post_template.add'
+    _title: 'Add post template'
+  requirements:
+    _permission: 'administer post entities'
+
+entity.post_template.edit_form:
```

Comment:
@semkulich Remove duplicates and not necessary routes. Do we have `/admin/config/post` or `/admin/config/post/post`, etc?